extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -1021,6 +1022,29 @@ describe('amp-a4a', () => {
         });
       });
     });
+
+    describe('protectFunctionWrapper', () => {
+      it('works properly with no error', () => {
+        let errorCalls = 0;
+        expect(protectFunctionWrapper_(name => {
+          return `hello ${name}`;
+        }, null, () => {errorCalls++;})('world')).to.equal('hello world');
+        expect(errorCalls).to.equal(0);
+      });
+
+      it('handles error properly', done => {
```

Comment:
I'm having a hard time figuring out how/whether this test works as intended (largely because of the `done` callback, whose semantics in synchronous code I can't figure out from the Chai API documentation and which should probably be avoided when possible). If `protectFunctionWrapper_`'s API is such that callers can rely on its return value, then I think it would be better to return an explicit value from the handler and assert that the top-level expression equals that. If not, then I think you should do something analogous to the `errorCalls` variable in the test above this one.

Also, probably-soon-to-be-irrelevant style nit: [parentheses are recommended around arrow function arguments.](https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions)